src/main/java/io/kestra/plugin/gcp/bigquery/AbstractBigquery.java

```diff
@@ -40,11 +40,11 @@ abstract public class AbstractBigquery extends AbstractTask {
 
     @Schema(
         title = "Automatic retry for retryable bigquery exceptions",
-        description = "Some exceptions (espacially rate limit) are not retried by default by BigQuery client, we use by " +
+        description = "Some exceptions (especially rate limit) are not retried by default by BigQuery client, we use by " +
```

Comment:
I am unsure of what this statement means. Can it be split into two statements at the comma?

Reply:
Me neither, it was there before ...
I don't think splitting it is a good idea as both part of the statement are strongly related.
I propose to not update the existing documentation too much, anyway some day, it should be reviewed in totality by someone ...